pinot-core/src/main/java/org/apache/pinot/core/data/readers/MultiplePinotSegmentRecordReader.java

```diff
@@ -64,7 +63,7 @@ public MultiplePinotSegmentRecordReader(@Nonnull List<File> indexDirs)
    * @param schema input schema that is a subset of the segment schema
    * @param sortOrder a list of column names that represent the sorting order
    */
-  public MultiplePinotSegmentRecordReader(@Nonnull List<File> indexDirs, @Nullable Schema schema,
+  public MultiplePinotSegmentRecordReader(List<File> indexDirs, @Nullable Schema schema,
```

Comment:
Did you leave `Schema` here for some backward compatibility or you missed this one? Or, because it makes sense to PinotSegmentReader to keep schema?

Reply:
I kept the schema for pinot segment backed record readers because the schema is extracted from the file.